src/extension.ts

```diff
@@ -76,13 +77,14 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
   const daemonStarted = await daemonStart();
 
   const providerLifecycle: extensionApi.ProviderLifecycle = {
-    status: () => crcStatus.getProviderStatus(),
-
+    status: () => {
+      return crcStatus.getProviderStatus();
+    },
     start: context => {
-      return startCrc(context.log);
+      return startCrc(context.log, this.telemetryLogger);
```

Comment:
I think you shouldn’t use `this.` to pass `telemetryLogger` in to `startCrc` function, as 
`telemetryLogger` defined as local variable, not like field of an object, so you passing `undefined`.